apps/coscrad-frontend/src/components/nav-menu/nav-menu-presenter.tsx

```diff
@@ -11,50 +11,45 @@ interface NavMenuPresenterProps {
 }
 
 export const NavMenuPresenter = ({ navItemInfos }: NavMenuPresenterProps): JSX.Element => {
-    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+    const [isOpen, setIsOpen] = useState(false);
 
-    const isOpen = !isNull(anchorEl);
-
-    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-        setAnchorEl(event.currentTarget);
+    const handleClick = () => {
+        setIsOpen(true);
     };
 
     const handleClose = () => {
-        setAnchorEl(null);
+        setIsOpen(false);
     };
 
     return (
-        <div data-testid="nav-menu">
-            <Box sx={{ minWidth: '100px' }}>
-                <IconButton
-                    id="basic-button"
-                    color="primary"
-                    aria-controls={isOpen ? 'basic-menu' : undefined}
-                    aria-haspopup="true"
-                    aria-expanded={isOpen ? 'true' : undefined}
-                    onClick={handleClick}
-                >
-                    <MenuRoundedIcon />
-                </IconButton>
-                <Menu
-                    id="basic-menu"
-                    anchorEl={anchorEl}
-                    open={isOpen}
-                    onClose={handleClose}
-                    MenuListProps={{
-                        'aria-labelledby': 'basic-button',
-                    }}
-                >
-                    {navItemInfos.map((navItemInfo) => (
-                        <NavMenuItem
-                            key={navItemInfo.label}
-                            navItemInfo={navItemInfo}
-                            handleClose={handleClose}
-                        />
-                    ))}
-                </Menu>
-                <AuthenticationButton />
+        <Box data-testid="nav-menu">
```

Comment:
This is great, not sure why I haven't tried this solution for the testid before.  No need for an extra div...